plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -353,268 +274,222 @@ class ServicesTable extends React.Component {
-        heading
-      },
-      {
-        className: this.getCellClasses,
-        headerClassName: this.getCellClasses,
-        prop: "status",
-        helpText: (
-          <Trans render="span">
-            At-a-glance overview of the global application or group state.{" "}
-            <a
-              href={MetadataStore.buildDocsURI(
-                "/deploying-services/task-handling"
-              )}
-              target="_blank"
-            >
-              Read more
-            </a>.
-          </Trans>
-        ),
-        render: this.renderStatus,
-        sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
-        heading
-      },
-      {
-        className: this.getCellClasses,
-        headerClassName: this.getCellClasses,
-        prop: "version",
-        render: this.renderVersion,
-        sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
-        heading
-      },
-      {
-        className: this.getCellClasses,
-        headerClassName: this.getCellClasses,
-        prop: "regions",
-        render: this.renderRegions,
-        sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
-        heading
-      },
-      {
-        className: this.getCellClasses,
-        headerClassName: this.getCellClasses,
-        prop: "instances",
-        render: this.renderInstances,
-        sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
-        heading
-      },
-      {
-        className: this.getCellClasses,
-        headerClassName: this.getCellClasses,
-        prop: "cpus",
-        render: this.renderStats,
-        sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
-        heading
-      },
-      {
-        className: this.getCellClasses,
-        headerClassName: this.getCellClasses,
-        prop: "mem",
-        render: this.renderStats,
-        sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
-        heading
-      },
-      {
-        className: this.getCellClasses,
-        headerClassName: this.getCellClasses,
-        prop: "disk",
-        render: this.renderStats,
-        sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
-        heading
-      },
-      {
-        className: this.getCellClasses,
-        headerClassName: this.getCellClasses,
-        prop: "gpus",
-        render: this.renderStats,
-        sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
-        heading
-      },
-      {
-        className: this.getCellClasses,
-        headerClassName: this.getCellClasses,
-        prop: "actions",
-        render: this.renderServiceActions,
-        sortable: false,
-        heading() {
-          return null;
-        }
-      }
-    ];
-  }
+          <Column
+            header={
+              <SortableHeaderCell
```

Comment:
This prop `textAlign="right"` will right-align it with the numbers. Same for GPU :)

Reply:
That makes the sorting caret not visible though. Is there a fix for that?

Reply:
I can't reproduce this in Firefox, Safari, or Chrome. Can you show me a screenshot?

Reply:
I mean that it's not visible when you are not hovering it (unlike other columns, where the caret remains even when not hovering it). I will send you a screenshot tomorrow.

Reply:
![peek 2019-02-08 10-40](https://user-images.githubusercontent.com/40791275/52467365-f0b09c00-2b8d-11e9-8284-37ec7bd1283f.gif)